yarn.lock

```diff
@@ -28733,7 +28749,16 @@ workerpool@6.2.1:
   resolved "https://registry.yarnpkg.com/workerpool/-/workerpool-6.2.1.tgz#46fc150c17d826b86a008e5a4508656777e9c343"
   integrity sha512-ILEIE97kDZvF9Wb9f6h5aXK4swSlKGUcOEGiIYb2OOu/IrDU9iwj0fD//SsA6E5ibwJxpEvhullJY4Sl4GcpAw==
 
-"wrap-ansi-cjs@npm:wrap-ansi@^7.0.0", wrap-ansi@7.0.0, wrap-ansi@^2.0.0, wrap-ansi@^5.1.0, wrap-ansi@^6.2.0, wrap-ansi@^7.0.0, wrap-ansi@^8.1.0, wrap-ansi@^9.0.0:
+"wrap-ansi-cjs@npm:wrap-ansi@^7.0.0":
```

Comment:
This seems a little strange to me, we have two entries resolving to the same dependency.  Any idea what's happening?

Reply:
I'm not sure why this is happening now. I also backported #203043, but they are not combined via `yarn_deduplicate` either.

Reply:
There is a resolution in `7.17` for `"wrap-ansi": "7.0.0"` causing the strangeness here. I manually deduped in 371884c6d58 and it sticks around through bootstrap, so going to move forward with this.